oss-ui/Makefile

```diff
@@ -0,0 +1,47 @@
+include ../metadata.mk
+
+PACKAGE_NAME = github.com/projectcalico/calico/oss-ui
+IMAGE_BUILD_MARKER = oss_ui_container-$(ARCH).created
+
+###############################################################################
+# include ../lib.Makefile
+#   Additions to EXTRA_DOCKER_ARGS need to happen before the include since
+#   that variable is evaluated when we declare DOCKER_RUN and siblings.
+###############################################################################
+include ../lib.Makefile
+
+# Configure variables used by ci/cd common targets from lib.Makefile.
+BUILD_IMAGES=goldmane
+
+BUILD_IMAGE_NAME=tigera/yarn-build
+QEMU_IMAGE ?= calico/qemu-user-static:latest
+
+DOCKER_RUN_RM:=docker run --rm \
+	--env CYPRESS_CACHE_FOLDER=/code/.cache/Cypress \
+	--env NPM_TOKEN=${NPM_TOKEN} \
+	--user $(shell id -u):$(shell id -g) \
+	-v $${PWD}:/code \
+	-w /code
+
+$(IMAGE_BUILD_MARKER): build
+	touch $@
+
+.PHONY: image build
+image: $(IMAGE_BUILD_MARKER)
+
+build-container:
+	docker buildx build --load --pull -t $(BUILD_IMAGE_NAME) -f docker-image/Dockerfile.build .
```

Comment:
Any reason not to use $(DOCKER_BUILD) from lib.Makefile?

Reply:
I _think_ this adds some different things specifically for building UI images, like crypress and yarn (not sure if it's in there).

Reply:
I misunderstood what you meant here, I'm switching to using the DOCKER_BUILD